plugins/woocommerce/src/Blocks/BlockTypes/Checkout.php

```diff
@@ -287,10 +287,14 @@ public function update_local_pickup_title( $post_id, $post ) {
 		$post_blocks = parse_blocks( $post->post_content );
 		$title       = $this->find_local_pickup_text_in_checkout_block( $post_blocks );
 
-		if ( $title ) {
-			$pickup_location_settings['title'] = $title;
-			update_option( 'woocommerce_pickup_location_settings', $pickup_location_settings );
+		// Set the title to be an empty string if it wasn't found, or is whitespace.
```

Comment:
The next if clause won't reset it to an empty string if it is only whitespace. We can either remove this part from the comment or change the if clause. Remember `empty()` will return true for string '0' :/

Reply:
Ah yeah I do want to change it. I want to keep `is_string` because even empty strings, or whitespace should be valid. If the merchant can enter whitespace and save the page, we should honour that. I think it's not our business to police what merchants input beyond what is safe. Thanks.